server/conf/routes

```diff
@@ -266,7 +266,8 @@ GET     /idcsRegister                controllers.LoginController.register(reques
 GET     /logout                      @org.pac4j.play.LogoutController.logout(request: Request)
 
 # Log out of the application through a separate endpoint that clears all sessions
-GET     /logoutAllSessions           controllers.LogoutAllSessionsController.index(request: Request)
+GET     /logoutAllSessions                       controllers.LogoutAllSessionsController.index(request: Request)
+GET     /logoutAllSessions/:email         controllers.LogoutAllSessionsController.logoutFromEmail(request: Request, email: String)
```

Comment:
Yeah, I don't like passing the email in the url. Can these be `POST`s instead? Why do we need to pass the email? Can we just lookup the user based on their session?

Reply:
Is there anything we can check to verify the calls to the endpoint are from where we expect them to be from? This endpoint shouldn't be called by just anyone and leaving it open worries me.

Reply:
I agree - working on adding some logic for that and will update the PR in a bit.

Reply:
Updated to a post with authority_id instead of email. I'll also plan to add a `validIpAddressesForLogout` setting, which can be configured with a list of allowed IP addresses as a follow up.